server/games/coop.py

```diff
@@ -20,3 +20,17 @@ def __init__(self, *args, **kwargs):
             'Difficulty': 3,
             'Expansion': 1
         })
+
+    async def _validate_game_mode_settings(self):
```

Comment:
Would it make sense to make this a "public" function?

Reply:
I've fixed that. Hopefully good to merge now.